docs/en/overview/Glossary.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: global
+title: Glossary
+group: Overview
+priority: 1
+---
+
+<!-- * Table of Contents
+{:toc} -->
+
+Glossary of Alluxio terminology
+
+<!-- what words should be included in glossary? best if can point to more detailed doc-->
+<!-- this is just an example, definitions taken from https://docs.alluxio.io/os/user/stable/en/overview/Architecture.html -->
+
+* Table of Contents
+{:toc}
+
+### Master
```

Comment:
`master` and `job master` should be considered as completely separate categories

`leading master` and `standby master` are subcategories of `master` but `job master` does not have any subcategories. `secondary master` is already effectively dead

also the entire concept of `master` and its corresponding `leading master` and `standby master` may go away very soon. so i actually wouldn't spend too much time until the 3.x architecture makes more progress

Reply:
okay, i'll ask beinan for more 3.x/dora related terms